src/vivarium_gates_iv_iron/components/observers.py

```diff
@@ -546,3 +546,160 @@ def metrics(self, index: pd.Index, metrics: Dict) -> Dict:
         metrics.update(self.counts)
         metrics.update(self.person_time)
         return metrics
+
+
+class HemoglobinObserver:
+    configuration_defaults = {
+        'metrics': {
+            'hemoglobin': {
+                'by_age': True,
+                'by_year': True,
+                'by_sex': False,
+            }
+        }
+    }
+
+    def __repr__(self):
+        return 'HemoglobinObserver()'
+
+    ##############
+    # Properties #
+    ##############
+
+    @property
+    def sub_components(self) -> List:
+        return []
+
+    @property
+    def name(self):
+        return 'hemoglobin_observer'
+
+    #################
+    # Setup methods #
+    #################
+
+    # noinspection PyAttributeOutsideInit
+    def setup(self, builder: Builder) -> None:
+        self.clock = builder.time.clock()
+        self.configuration = builder.configuration.metrics.hemoglobin
+        self.start_time = get_time_stamp(builder.configuration.time.start)
+        self.hemoglobin = builder.value.get_value("hemoglobin.exposure")
+        self.step_size = builder.time.step_size()
```

Comment:
I don't see where you use clock, start_time, or step_size. If I'm correctly reading this, you don't need any of them